src/client/components/PageLayout/Toolbar/Status/StatusConfirm/UserList/UserList.tsx

```diff
@@ -0,0 +1,66 @@
+import './UserList.scss'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { User, Users } from 'meta/user'
+
+import { useCycle } from 'client/store/assessment'
+import { useCountryRouteParams } from 'client/hooks/useRouteParams'
+import { DataCell, DataGrid } from 'client/components/DataGrid'
+
+interface Props {
+  users: Array<User>
+}
+
+interface CellProps {
+  user: User
+}
+
+const NameCell: React.FC<CellProps> = ({ user }) => <span>{Users.getFullName(user)}</span>
```

Comment:
Is it preferred in this case separate file with `(props: Props)` for all Cell components?